### DISCUSSION TITLE
Add advanced charts options (mixed charts, labels & more)

### DISCUSSION THREAD

**Discussion by DanWearr** [mdbootstrap/TW-Elements#1534] [Share ideas & request features]
These are the features that would be awesome to add to current Tailwind Elements charts:

- An option to create mixed charts - i.e. bar chart & line chart together
![image](https://user-images.githubusercontent.com/129263992/228460357-cd380bb8-7074-4f0a-9e37-fd21213b16de.png)

- Double axis - an additional axis on the right side for the second chart
![image](https://user-images.githubusercontent.com/129263992/228460484-9e121bbd-f4fb-4af0-bdae-be82c09bb56c.png)

- Different values for data labels - an option to display values on the chart, that could be customized to be different than values in the tooltips already on the chart. I.e. pie chart parts show percentages & tooltips show number of orders:
![image](https://user-images.githubusercontent.com/129263992/228460984-f445fe3c-926b-41f5-8787-0aaccae26972.png)

- Double bar charts (different sets of data in time)
- Double line charts
- Horizontal bar charts

I hope you will take that into consideration. Thanks for your amazing work so far!

**Comment by filipkappa** [comment]
Thanks for submitting this idea!

It gathered so much traction that we have added a task for this feature: https://github.com/mdbootstrap/Tailwind-Elements/issues/1555

You can track our progress live here:
https://tailwind-elements.com/docs/standard/getting-started/changelog/
